Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx

```diff
@@ -0,0 +1,259 @@
+  const { templates, onDismiss, onNext } = props;
+  const [currentTemplate, setCurrentTemplate] = useState('');
+  const [emptyBotKey, setEmptyBotKey] = useState('');
+  const [selectedFeed, setSelectedFeed] = useState<{ props: IPivotItemProps }>({ props: { itemKey: csharpFeedKey } });
+  const readMe = useRecoilValue(selectedTemplateReadMeState);
+
+  const selectedTemplate = useMemo(() => {
+    return new Selection({
+      onSelectionChanged: () => {
+        const t = selectedTemplate.getSelection()[0] as BotTemplate;
+        if (t) {
+          setCurrentTemplate(t.id);
+        }
+      },
+    });
+  }, []);
+
+  const handleJumpToNext = () => {
+    let routeToTemplate = emptyBotKey;
+    if (option === optionKeys.createFromTemplate) {
+      routeToTemplate = currentTemplate;
+    }
+
+    if (option === optionKeys.createFromQnA) {
+      routeToTemplate = QnABotTemplateId;
+    }
+
+    if (props.location && props.location.search) {
+      routeToTemplate += props.location.search;
+    }
+
+    TelemetryClient.track('CreateNewBotProjectNextButton', { template: routeToTemplate });
+
+    onNext(routeToTemplate);
+  };
+
+  const tableColumns = [
+    {
+      key: 'name',
+      name: formatMessage('Name'),
+      fieldName: 'name',
+      minWidth: 150,
+      maxWidth: 200,
+      isResizable: !disabled,
+      data: 'string',
+      styles: rowTitle(disabled),
+      onRender: (item) => (
+        <div data-is-focusable css={tableCell}>
+          <div css={content} tabIndex={-1}>
+            {item.name}
+          </div>
+        </div>
+      ),
+    },
+  ];
+
+  const onRenderRow = (props) => {
+    if (!props) return null;
+    return (
+      <DetailsRow {...props} data-testid={props.item.id} styles={rowDetails(disabled)} tabIndex={props.itemIndex} />
+    );
+  };
+
+  useEffect(() => {
+    if (templates.length > 1) {
+      const emptyBotTemplate = find(templates, ['id', EmptyBotTemplateId]);
+      if (emptyBotTemplate) {
+        setCurrentTemplate(emptyBotTemplate.id);
+        setEmptyBotKey(emptyBotTemplate.id);
+      }
+    }
+  }, [templates]);
+
+  useEffect(() => {
+    if (selectedFeed?.props?.itemKey) {
+      props.fetchTemplates([feedDictionary[selectedFeed.props.itemKey]]);
+    }
+  }, [selectedFeed]);
+
+  useEffect(() => {
+    if (currentTemplate) {
+      props.fetchReadMe(currentTemplate);
+    }
+  }, [currentTemplate]);
+
+  return (
+    <Fragment>
+      <DialogWrapper
+        isOpen
+        {...DialogCreationCopy.CREATE_NEW_BOT_V2}
+        dialogType={DialogTypes.CreateFlow}
+        onDismiss={onDismiss}
+      >
+        <Pivot
+          defaultSelectedKey={csharpFeedKey}
+          onLinkClick={(item) => {
+            if (item) {
+              setSelectedFeed(item);
+            }
+          }}
+        >
+          <PivotItem headerText="C#" itemKey={csharpFeedKey}></PivotItem>
+          <PivotItem headerText="Typescript" itemKey={nodeFeedKey}></PivotItem>
```

Comment:
Before we ship, we can kill this pivot since we will not have Typescript templates. Til then let's leave it to test underyling features.

Reply:
Okay I will disable it as an option so users wont be able to click it

Reply:
Hmm it seems that UI element is not disableable. I have removed it for now so there is only the C# tab. Will iterate with design for a better short term solution if needed. (iterating with design task is created and will be handled in its own PR)